doc/lua_api.txt

```diff
@@ -5465,6 +5474,14 @@ This is basically a reference to a C++ `ServerActiveObject`
 * `get_player_name()`: returns `""` if is not a player
 * `get_player_velocity()`: returns `nil` if is not a player, otherwise a
   table {x, y, z} representing the player's instantaneous velocity in nodes/s
+* `add_player_velocity(vel)`
+    * Adds to player velocity, this happens client-side and only once.
+    * Does not apply during free_move.
+    * Note that since the player speed is normalized at each move step,
+      increasing e.g. Y velocity beyond what would usually be achieved
+      (see: physics overrides) will cause existing X/Z velocity to be reduced.
```

Comment:
Does this mean that `add_player_velocity` cannot be used to knock a player back with speed significantly higher than walk speed? Also how does this interact with anticheat?

Reply:
> Does this mean [...]

Yes

> Also how does this interact with anticheat?

It works.